views/login.tpl.html

```diff
@@ -1,55 +1,183 @@
+
+      <!-- WebAuthn Success/Error Messages -->
+      <div id="webauthn-error" class="mb-4 alert alert-error" style="display: none;"></div>
+      <div id="webauthn-success" class="mb-4 alert alert-success" style="display: none;"></div>
+
+      <!-- Login Form -->
+      <form action="login" method="post" id="login-form">
+        <div class="mb-4">
+          <input class="input-default" type="text" id="username" autocomplete="username" name="username"
+            placeholder="Username" minlength="1" required autofocus>
+        </div>
+        <div class="mb-4">
+          <input class="input-default" type="password" id="password" autocomplete="current-password" name="password"
+            placeholder="Password{{ if .WebAuthnEnabled }} (or use passkey){{ end }}" minlength="6">
         </div>
-        <form action="login" method="post">
-            <div class="mb-4">
-                <input class="input-default"
-                       type="text" id="username" autocomplete="username"
-                       name="username" placeholder="Username" minlength="1" required autofocus>
-            </div>
-            <div class="mb-4">
-                <input class="input-default"
-                       type="password" id="password" autocomplete="current-password"
-                       name="password" placeholder="Password" minlength="6" required>
-            </div>
-            <div class="flex justify-between items-center">
-                <a href="reset-password" class="text-muted text-sm">
-                    Forgot password?
-                </a>
-                <div class="flex space-x-2">
-                    {{ if .AllowSignup }}
-                    <a href="signup">
-                        <button type="button" class="btn-default">Sign up</button>
-                    </a>
-                    {{ else }}
-                    <a title="The administrator of this instance has disabled sign up.">
-                        <button type="button" class="btn-disabled" disabled> Sign up </button>
-                    </a>
-                    {{ end }}
-                    <button type="submit" class="btn-primary">Log in</button>
-                </div>
-            </div>
-        </form>
+        <div class="flex items-center justify-between">
+          <a href="reset-password" class="text-sm text-muted">
+            Forgot password?
+          </a>
+          <div class="flex space-x-2">
+            {{ if .AllowSignup }}
+            <a href="signup">
+              <button type="button" class="btn-default">Sign up</button>
+            </a>
+            {{ else }}
+            <a title="The administrator of this instance has disabled sign up.">
+              <button type="button" class="btn-disabled" disabled> Sign up </button>
+            </a>
+            {{ end }}
+            <button type="submit" class="btn-primary" id="login-btn">
+              <svg class="inline w-4 h-4 mr-2" fill="currentColor" viewBox="0 0 20 20">
```

Comment:
Why? Despite this icon being inappropriate and despite the fact that don't use icons on these types of buttons anywhere else, icons shall always be included using the _iconify_ classes instead of raw SVGs. Please remove.